Watchman.Integrations/Quickchart/QuickchartService.cs

```diff
@@ -15,13 +14,9 @@ public string GetImage(Chart chart)
             var datasetValues = chart.Data.Data.Select(x => x.ToString()).Aggregate((a, b) => a + "," + b);
             var url = baseUrl + $"?c={{type:'{chart.Type}',data:{{labels:[{labels}], datasets:[{{label:'{chart.Data.Label}',data:[{datasetValues}]}}]}}}}" + "&backgroundColor=white";
 
-            using (var client = new WebClient())
-            {
-                client.DownloadFile(url, "statistics.png");
-                return "statistics.png";
-            }
+            using var client = new WebClient();
```

Comment:
tak właściwie to czemu tu jest WebClient a nie HttpClient? 
może był jakiś powód o którym nie pamiętam 

plus ciekawą biblioteką do obsługi http jest RestSharp

Reply:
Ja tego nie wiem, bo nigdy przy tym nic nie robiłem

Reply:
HttpClient będzie ok

Reply:
Już chyba wiem dlaczego
![image](https://user-images.githubusercontent.com/35143425/77193197-2688b980-6ade-11ea-9eea-e85a3a69d43c.png)

Reply:
może zostać WebClient
a tak poza tym, w wolnym czasie możesz poszukać czy da sie przez Discord.NET wysyłać pliki jako stream a nie ścieżke do pliku 
sporo by to ułatwiło 

jeśli nie, to mamy kolejny argument do napisania własnej implementacji do API discorda